bi-att-flow-dev/basic/trainer.py

```diff
@@ -46,13 +46,17 @@ def __init__(self, config, models):
         self.models = models
         losses = []
         grads_list = []
+        print(self.var_list) # TODO: Check is this should not be None?
         for gpu_idx, model in enumerate(models):
             with tf.name_scope("grads_{}".format(gpu_idx)), tf.device("/{}:{}".format(config.device_type, gpu_idx)):
                 loss = model.get_loss()
                 grads = self.opt.compute_gradients(loss, var_list=self.var_list)
                 losses.append(loss)
                 grads_list.append(grads)
 
+        print(losses)
```

Comment:
what is the fix that you made in this file

Reply:
Oh nothing actually. To fix this I needed to fix the model actually.